Cargo.toml

```diff
@@ -44,9 +44,9 @@ unstable-stream = []
 __internal_proxy_sys_no_cache = []
 
 [dependencies]
-http = "0.1.15"
+http = "0.1.19"
 url = "2.1"
-bytes = "0.4"
+bytes = "0.4.12"
```

Comment:
Is there a reason to update this specifically? Cargo will use the latest patch version by default.

Reply:
No special reason, you're right.
I can undo it.